README.md

```diff
@@ -1,2 +1,76 @@
-# puppetlabs-powershell_task_helper
-A helper library for Puppet Tasks written in PowerShell
+# powershell_task_helper
+
+A helper library for Puppet Tasks written in PowerShell.
+
+## Table of Contents
+
+1. [Description](#description)
+1. [Setup - The basics of getting started with powershell_task_helper](#setup)
+    * [Beginning with powershell_task_helper](#beginning-with-powershell_task_helper)
+1. [Usage - Configuration options and additional functionality](#usage)
+1. [Limitations - OS compatibility, etc.](#limitations)
+1. [Development - Guide for contributing to the module](#development)
+
+## Description
+
+A PowerShell helper library for use by [Puppet Tasks](https://puppet.com/docs/bolt/latest/writing_tasks.html). It provides a function that handles error generation, simplifies JSON input and output, and makes testing your task easier. It requires Bolt >= 2.4.
+
+## Setup
+
+### Beginning with powershell_task_helper
+
+To use this library, include this module in a `Puppetfile`
+
+```ruby
+mod 'puppetlabs-powershell_task_helper'
+```
+
+Add it to your [task metadata](https://puppet.com/docs/bolt/1.x/writing_tasks.html#concept-677)
+
+```json
+{
+  "files": ["powershell_task_helper/files/BoltPwshHelper/"],
+  "input_method": "powershell"
+}
+```
+
+## Usage
+
+When writing your task include the library in your script by dot-sourcing the `task_helepr.ps1` file
```

Comment:
```suggestion
When writing your task include the library in your script by dot-sourcing the `task_helepr.ps1` file
```

Since Bolt is getting updated to auto-load modules, I'm assuming this is outdated and instead should say to add `Import-Module BoltPwshHelper` for PS 3?